src/styles/blocks/blocks.scss

```diff
@@ -0,0 +1,30 @@
+.page {
+  font-family: Arial, sans-serif;
+  font-size: 100px;
+}
+
+.container {
+  display: grid;
+  grid-template-columns: repeat(auto-fill,
+  #{"minmax(max(300px, (33.3%)), 1fr)"});
```

Comment:
Interesting solution👍

Reply:
But too complicated, could be much easier.